src/test/regress/expected/view_propagation.out

```diff
@@ -359,13 +359,93 @@ ERROR:  cannot change name of view column "id" to "a"
 HINT:  Use ALTER VIEW ... RENAME COLUMN ... to change name of view column instead.
 CREATE OR REPLACE VIEW view_for_unsup_commands AS SELECT id FROM table_to_test_unsup_view;
 ERROR:  cannot drop columns from view
+-- ALTER VIEW PROPAGATION
+CREATE TABLE alter_view_table(id int, val1 text);
+SELECT create_distributed_table('alter_view_table','id');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE VIEW alter_view_1 AS SELECT * FROM alter_view_table;
+-- Set/drop default value is not supported by Citus
+ALTER VIEW alter_view_1 ALTER COLUMN val1 SET DEFAULT random()::text;
+ERROR:  Citus doesn't support setting or resetting default values for a column of view
+ALTER TABLE alter_view_1 ALTER COLUMN val1 SET DEFAULT random()::text;
+ERROR:  Citus doesn't support setting or resetting default values for a column of view
+ALTER VIEW alter_view_1 ALTER COLUMN val1 DROP DEFAULT;
+ERROR:  Citus doesn't support setting or resetting default values for a column of view
+ALTER TABLE alter_view_1 ALTER COLUMN val1 DROP DEFAULT;
+ERROR:  Citus doesn't support setting or resetting default values for a column of view
+-- Set/reset options view alter view/alter table commands
+ALTER VIEW alter_view_1 SET (check_option=cascaded);
+ALTER VIEW alter_view_1 SET (security_barrier);
+ALTER VIEW alter_view_1 SET (check_option=cascaded, security_barrier);
+ALTER VIEW alter_view_1 SET (check_option=cascaded, security_barrier = true);
+ALTER TABLE alter_view_1 SET (check_option=cascaded);
+ALTER TABLE alter_view_1 SET (security_barrier);
+ALTER TABLE alter_view_1 SET (check_option=cascaded, security_barrier);
+ALTER TABLE alter_view_1 SET (check_option=cascaded, security_barrier = true);
+\d+ alter_view_1
+                    View "view_prop_schema.alter_view_1"
+ Column |  Type   | Collation | Nullable | Default | Storage  | Description
```

Comment:
as I also noted in the other PR, `\d` may change outputs per release, so if possible we avoid and query pg_class (such that we wouldn't end up adding new output files if a version changes the description)